code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -14,7 +14,7 @@
 
 /obj/item/projectile/bullet/c38
 	name = ".38 bullet"
-	damage = 25
+	damage = 30

```

Comment:
seems kind of like a stealth change

Reply:
Real' stealthy the way I asked people ahead of time and specifically noted it in the commit.

Reply:
not in PR body or changelog or anywhere but the commit and in a PR that's primarily for cosmetic changes

Reply:
Sorry that I didn't update the changelog after all this shit. I just want this to be over with, the merge conflict was a headache.

Reply:
revert this and put it in a separate PR then i'll merge this PR

```suggestion
	damage = 25
```